<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DESForm</class>
 <widget class="QMainWindow" name="DESForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DES Encoder</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTableWidget" name="tblResult">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string>File</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Action</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vlButtons">
      <item>
       <widget class="QPushButton" name="btnEncrypt">
        <property name="text">
         <string>Encrypt File...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDecrypt">
        <property name="text">
         <string>Decrypt File...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>btnEncrypt</tabstop>
  <tabstop>btnDecrypt</tabstop>
  <tabstop>tblResult</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
